### DISCUSSION TITLE
How do you convert Results into a Pandas DataFrame?

### DISCUSSION THREAD

**Discussion by cirobressiani** [DaveSkender/Stock.Indicators#1203] [Q&A]
Hi,
How to transform the RSI results in a Pandas Dataframe instead of print it? For exemple:

# calculate RSI(14)
results = indicators.get_rsi(quotes, 14)

I would like to see the 'results' variable in a dataframe format to manipulate it, instead of print it to visualize the RSI data.

Thanks

**Comment by m-elek** [comment]
df["rsi"]  = [ row.rsi for row in results ]
